src/Test.UnitTests.Sarif.WorkItems/SarifWorkItemModelTests.cs

```diff
@@ -36,7 +37,23 @@ public void SarifWorkItemModel_PopulatesADODescription()
             var workItemModel = new SarifWorkItemModel(sarifLog, context);
             workItemModel.BodyOrDescription.Should().NotBeNullOrEmpty();
             workItemModel.BodyOrDescription.Should().Contain(nameof(TestData.TestToolName));
-            workItemModel.BodyOrDescription.Should().Contain("To see result details, please visit the Scans tab of this bug, or the attached SARIF log.");
+            workItemModel.BodyOrDescription.Should().Contain(sarifLog.Runs[0].VersionControlProvenance[0].RepositoryUri.OriginalString);
+            workItemModel.BodyOrDescription.Should().Contain("Visual Studio SARIF add-in.");
+        }
+
+        [Fact]
+        public void SarifWorkItemModel_MultipleToolsADODescription()
```

Comment:
>ADO [](start = 52, length = 3)

.NET style guidelines: acronyms have only the first letter capitalized, so `Ado`. (Same on previous test.)

As for the test name, a style that I advocate (and I've persuaded Michael that it's a good one) is that the test name describes the behavior: `Car_CanStart`. Sometimes there is a condition in the name `Car_WhenOutOfGas_CannotStart`. This general naming convention comes from Behavior-Driven Development (BDD), FWIW.

So here, perhaps `SarifWorkItemModel_IncorporatesMultipleToolNamesIntoAdoDescription`.

Reply:
Michael did mention this, I haven't internalized it yet, but agree with the convention.
Stealing your name for this one.

---
In reply to: [398221599](https://github.com/microsoft/sarif-sdk/pull/1823#discussion_r398221599) [](ancestors = 398221599)